Package.swift

```diff
@@ -58,7 +58,7 @@ var package = Package(
     .package(url: "https://github.com/apple/swift-algorithms", from: "1.0.0"),
     .package(url: "https://github.com/devxoul/Then", from: "2.7.0"),
     .package(url: "https://github.com/mkrd/Swift-BigInt", from: "2.0.0"),
-    .package(url: "https://github.com/GuardianFirewall/GuardianConnect", exact: "1.8.1"),
+    .package(url: "https://github.com/GuardianFirewall/GuardianConnect", exact: "1.8.3"),
```

Comment:
Can we put this in a separate PR with a small description of the problem/solution for the future :)